libgalois/include/katana/PropertyGraph.h

```diff
@@ -125,6 +127,13 @@ class KATANA_EXPORT GraphTopology {
     return MakeStandardRange<node_iterator>(begin, end);
   }
 
+  nodes_range all_nodes() const noexcept {
```

Comment:
Why not just `nodes`?

Reply:
My intention is to clarify with the name that you will iterate over all nodes. There is nodes(beg, end) and I wanted to differentiate from that instead of overload. Similarly edges(node_id) vs edges() (as in all_edges()) are different, so I didn't want them to get confused with each other.